src/main/java/hudson/plugins/copyartifact/StatusBuildSelector.java

```diff
@@ -48,7 +48,7 @@ public boolean isStable() {
 
     @Override
     public boolean isSelectable(Run<?,?> run, EnvVars env) {
-        return run.getResult().isBetterOrEqualTo(isStable() ? Result.SUCCESS : Result.UNSTABLE);
+        return isBuildResultBetterOrEqualTo(run, isStable() ? Result.SUCCESS : Result.UNSTABLE);
```

Comment:
```
    <BugInstance type='NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE'
        priority='Normal'
        category='STYLE'
        message='Possible null pointer dereference in hudson.plugins.copyartifact.StatusBuildSelector.isSelectable(Run, EnvVars) due to return value of called method'
        lineNumber='51'/>
```

Actually, `isSelectable` is expected to be called for only completed builds and `run.getResult()` should not be null.
(Though, wrapped with a helper method with null checks)